modular_skyrat/code/modules/mob/living/carbon/human/species_types/speciessnowflake.dm

```diff
@@ -0,0 +1,95 @@
+
+/datum/species/insect/moth
+	screamsounds = list('modular_citadel/sound/voice/scream_moth.ogg')
+	fluff_desc = "Moth people seem to be a splinter of insect people species, and are very similar in biology and appearance."
+
+/datum/species/human
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Humans come from the Sol system, and are the commanding species of both NanoTrasen and SolGov."
+
+/datum/species/angel
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Not much is know about angels, but they are most commonly associated with christian mythology."
+
+/datum/species/corporate
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Corporate robots are simply advanced corporate androids employed by NT. Most of the time, they are high ranking CC officers."
+
+/datum/species/dullahan
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Not much is known about the dullahan, but they sure know how to make someone lose their minds."
+
+/datum/species/dwarf
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Dwarves are very similar to humans in biology, but they seem to have completely split from them as a species."
+
+/datum/species/mammal
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Anthropomorphs encompass a lot of the know sapient species by SolGov, and their backgrounds always vary wildly."
+
+/datum/species/jelly
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Slimepeople are the result of scientifical experimentation on slimes, which NanoTrasen has heavily invested in. At the current day, most slimepeople are simply consumers of brands of slime genemods sold by NanoTrasen."
+
+/datum/species/plasmaman
+	screamsounds = list('modular_citadel/sound/voice/scream_skeleton.ogg')
+	fluff_desc = "Plasmamen are the result of catastrophic experimentation with plasma. They are very fragile and in most circumstances require external support to survive, and for this reason are considered most of the time simply cheap workforce."
+
+/datum/species/pod
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Podpeople are simply plantified clones of other sapient species - No one knows where podplants were found for sure, but podpeople have since formed splinter groups about glorifying and preserving nature, coexisting with advanced technology."
```

Comment:
"Podpeople are a race of sapient humanoid plants, oftentimes the result of genetic experimentation or botanical research gone awry. While most podpeople are the result of experimental revival techniques involving a species of large cabbage, plant-based lifeforms can take a wide variety of shapes and forms within the galaxy.